<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWindow</class>
 <widget class="QMainWindow" name="AuthWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(24, 24, 24);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loginOptionButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40);
color:white;</string>
    </property>
    <property name="text">
     <string>Logowanie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerOptionButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40);
color:white;</string>
    </property>
    <property name="text">
     <string>Rejestracja</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="authframe">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>441</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Login:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Haslo:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(40, 40, 40);
color:white;</string>
      </property>
      <property name="text">
       <string>Zaloguj</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginData">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>231</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 25px;
background-color: rgb(40, 40, 40); 
color: white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordData">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>231</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 25px;
background-color: rgb(40, 40, 40); 
color: white;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Login:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Haslo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Imie:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Nazwisko:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>210</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Nick:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="registerButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(40, 40, 40);
color:white;</string>
      </property>
      <property name="text">
       <string>Zarejestruj</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerLoginData">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>231</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 25px;
background-color: rgb(40, 40, 40); 
color: white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerPasswordData">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>231</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 25px;
background-color: rgb(40, 40, 40); 
color: white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerNameData">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>231</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 25px;
background-color: rgb(40, 40, 40); 
color: white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerSurnameData">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>231</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 25px;
background-color: rgb(40, 40, 40); 
color: white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerNicknameData">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>210</y>
        <width>231</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 25px;
background-color: rgb(40, 40, 40); 
color: white;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
